From 2fc539daa0e51a280f36c3d3558f15856bc217fd Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <jan.steffens@gmail.com>
Date: Fri, 17 Feb 2017 23:22:59 +0100
Subject: [PATCH 1/2] lua-factory: Remove bad prototype for html_entity_hash

gperf 3.1 changes the type of the len argument for the generated
function to size_t, causing a conflict.
---
 src/lua-factory/lua-library/htmlentity.gperf | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/lua-factory/lua-library/htmlentity.gperf b/src/lua-factory/lua-library/htmlentity.gperf
index cbd326c4f56f3e2d..096bf3e39b106eda 100644
--- a/src/lua-factory/lua-library/htmlentity.gperf
+++ b/src/lua-factory/lua-library/htmlentity.gperf
@@ -35,8 +35,6 @@ struct _EntityEntry {
 	const gchar *name;
 	gulong value;
 };
-struct _EntityEntry *
-html_entity_hash (register const char *str, register unsigned int len);
 %%
 Aacute, 193
 aacute, 225
-- 
2.11.1

